lib/index.js

```diff
@@ -154,8 +155,8 @@ api.insert = brCallbackify(async ({actor, account, meta}) => {
   // validate resource roles (ensure actor is permitted to delegate the
   // roles specified in the account meta)
   if(meta.sysResourceRole.length > 0) {
-    await validateCapabilityDelegation(
-      {actor, resourceRoles: meta.sysResourceRole});
+    const caps = {actor, resourceRoles: meta.sysResourceRole};
+    await validateCapabilityDelegation(caps);
```

Comment:
This is confusing. Why the change?

Reply:
this was causing an eslint parentheses error with line breaks.

Reply:
We'll want to fix eslint then ... this should be ok and we do it a lot. Also, this was confusing because the variable name was `caps` ... and the above isn't some kind of "capabilities" object, it's two named parameters to a function. If anything I guess it could be called `options`. That being said, I think we should fix eslint so this is ok -- and if there's no option to make it ok, we should disable whatever is making it not ok.

Reply:
simply put what you want I understand, but the vanilla eslint rule for func-parens just doesn't handle the cases correctly. There has to be a plugin though that does handle it. maybe unicorn has it? we should just remove this rule for the time being until we can find a proper plugin that understands our needs and just make func parens written in an understandable way.

Reply:
we could try: { "minItems": value }
but the issue here is that sometimes we only have 2 paramaters that happen to have really long lengths.